src/configAPIClient.ts

```diff
@@ -1,13 +1,37 @@
-export default function ConfigAPIClient() {
-    this.getSDKConfiguration = function(
+import { DataPlanConfig } from '@mparticle/web-sdk';
+import { KitConfigs, MParticleWebSDK, SDKInitConfig } from './sdkRuntimeModels';
+
+export type SDKInitFunction = (
```

Comment:
The name of this type is misleading.  It's not really the `SDKInitFunction`, since that does't pass the mpInstance.  Kind of wordy, but I'd rename this to `CompleteSDKInitFunction`